skimage/measure/_ccomp.pyx

```diff
@@ -357,104 +342,14 @@ def undo_reshape_array(arr, swaps):
     return reshaped
 
 
-# Connected components search as described in Fiorio et al.
-def label(input, neighbors=None, background=0, return_num=False,
+
+def label(input, neighbors=None, background=None, return_num=False,
```

Comment:
Please, rename this to `_label_cython` and change the corresponding import in `.py`.